ydb_dbapi/cursors.py

```diff
@@ -191,16 +217,18 @@ def execute(
                 tx_context=self._tx_context, query=query, parameters=parameters
             )
         else:
-            self._stream = self._execute_generic_query(
+            self._stream = self._execute_session_query(
                 query=query, parameters=parameters
             )
 
         self._begin_query()
-
         self._scroll_stream(replace_current=False)
 
-    async def executemany(self) -> None:
-        pass
+    def executemany(
+        self, query: str, seq_of_parameters: Sequence[ParametersType]
+    ) -> None:
+        for parameters in seq_of_parameters:
+            self.execute(query, parameters)
```

Comment:
**question (non-blocking)**: Could we optimize this using preparation of query or just concat queries together?